packages/cdktf-cli/lib/dependencies/dependency-manager.ts

```diff
@@ -241,10 +248,14 @@ export class DependencyManager {
   /**
    * Converts an NPM package name of a pre-built provider package to the name in the target language
    */
-  private convertPackageName(name: string): string {
+  private convertPackageName(name: string, repository: string): string {
     const providerName = name.replace("@cdktf/provider-", "");
     switch (this.targetLanguage) {
       case Language.GO: // e.g. github.com/hashicorp/cdktf-provider-opentelekomcloud-go/opentelekomcloud
```

Comment:
While we're at it we could update the comment 😁
```suggestion
      case Language.GO: // e.g. github.com/cdktf/cdktf-provider-opentelekomcloud-go/opentelekomcloud
```

Reply:
(even if it is not yet true, but it soon will be)

Reply:
:/ Github didn't let me merge these in a single commit, so decided to do it manually.

Reply:
No worries!